src/python/pants/option/arg_splitter_test.py

```diff
@@ -228,22 +212,6 @@ def test_passthru_args(splitter: ArgSplitter) -> None:
     )
 
 
-def test_subsystem_flags(splitter: ArgSplitter) -> None:
```

Comment:
We no longer use ArgSplitter to extract flags (we only use it to figure out goals and specs), so this test now just recapitulates other tests in this file.